monitoring/defender/autotask/on_module_factory_events.js

```diff
@@ -1184,27 +1241,46 @@ exports.handler = async function (event) {
   );
   if (event && event.request && event.request.body) {
     // variables from autotask creation
+    const etherscanUrl = "{{etherscanUrl}}";
+    const etherscanApiKey = "{{etherscanApiKey}}";
     const rpcUrl = "{{rpcUrl}}";
     const discordWebHookUrl = "{{discordWebHookUrl}}";
+    const mastercopy = "{{mastercopyAddress}}";
+    const blockHash = parseInt(event.request.body.blockNumber, 16);
+    const chainId = event.request.body.sentinel.chainId;
     const transaction = event.request.body.transaction;
     const logs = transaction.logs;
     const txHash = transaction.transactionHash;
     const provider = new ethers.providers.JsonRpcProvider(rpcUrl);
-    const chainId = event.request.body.sentinel.chainId;
+    const currentBlock = await provider.getBlock();
+    const templateLogs = await getRealityEthLogs(
+      etherscanUrl,
+      etherscanApiKey,
+      chainId,
+      blockHash,
+      currentBlock.number
```

Comment:
`fromBlock` is a hash, while `toBlock` is a number, If this is correct, I think we have to update the function doc. The function comment says it has to be a number 🤔